<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupromodoro">
    <property name="title">
     <string>promodoro</string>
    </property>
    <addaction name="TryPromodoro"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Epub_to_Mobi</string>
    </property>
    <addaction name="actionChoose_Epub_File"/>
   </widget>
   <widget class="QMenu" name="menuHelo">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_Book"/>
   </widget>
   <addaction name="menupromodoro"/>
   <addaction name="menuHelp"/>
   <addaction name="menuHelo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="TryPromodoro">
   <property name="text">
    <string>Try it!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionHelp_Book">
   <property name="text">
    <string>Help Book</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionChoose_Epub_File">
   <property name="text">
    <string>to conver file!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
